meteor/client/ui/PieceIcons/Renderers/UnknownInput.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+
+// @todo: use colours from the scss
```

Comment:
Is there a reason this can't be done now?

Reply:
Actually, I think we're already using colors from SCSS, so I think this is a leftover comment from the file this was based on.